### PR TITLE
Add Spec Run To CI

### PR DESCRIPTION
**Pull request by orangewolf**
Super Green

### CODE REVIEW

**Comment by orangewolf** [comment]
I thought automated specs would be a good PR #200 

@bkiahstroud I renamed the Rubocop to be lint / rubocop since "Kind of thing / Thing" made more sense to me. Let me know if you want that switched back.

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Rubocop
+name: Lint
```
Comment (bkiahstroud) [inline .github/workflows/lint.yml:1]: Extra whitespace.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        id: ruby
+        with:
+          ruby-version: 2.6.5
+          cache-available: ${{ steps.cache.outputs.cache-hit == 'true' }}
+
+      - name: Setup sqlite3
+        run: sudo apt-get install libsqlite3-dev
+
+      - name: Install dependencies
+        run: bundle install
```
Comment (bkiahstroud) [inline .github/workflows/test.yml:33]: Do we need to bundle all the gems to run the specs? If not, should we flesh out the `:test` group in the Gemfile to allow us to target those specifically?

Comment (bkiahstroud) [reply .github/workflows/test.yml:33]: Bundling all the gems took 4 and a half minutes on the last run.

Comment (orangewolf) [reply .github/workflows/test.yml:33]: yes, you need them all, the stuff in dev/test that is slow is the stuff for running the specs.

File: .github/workflows/lint.yml (hunk 1 above)
Comment (orangewolf) [reply .github/workflows/lint.yml]: fixed

**Comment by orangewolf** [comment]
yes you need the .gitkeep - it forces the directory to exist as git does not keep track of directories, only stuff in directories.

**Pull request merged**